zone/jt

```diff
@@ -0,0 +1,2 @@
+jt    IN      A       2.51.170.185:1234 (Port 80 cannot be opened on my server)
```

Comment:
```suggestion
jt    IN      A       2.51.170.185
```
Because this configuration will be added to the server automatically, we can't put comments in these files. Also, A records don't support ports. See: http://www.zytrax.com/books/dns/ch8/a.html